browser/brave_content_browser_client.cc

```diff
@@ -589,6 +596,20 @@ bool BraveContentBrowserClient::CanCreateWindow(
       container_type, target_url, referrer, frame_name, disposition, features,
       user_gesture, opener_suppressed, no_javascript_access);
 
+#if BUILDFLAG(ENABLE_REQUEST_OTR)
+  // If the user has requested going off-the-record in this tab, don't allow
+  // opening new windows via script
+  if (content::WebContents* web_contents =
```

Comment:
doesn't this need a feature flag guard as well?

Reply:
actually it's probably better to check for null `request_otr_tab_storage` instead of checking the feature directly here (we should have that check either way)

Reply:
We have that check. It's wrapped in an `if` statement already.